docs/viz.md

```diff
@@ -0,0 +1,320 @@
+# Visualization and Analysis
+
+<!--*********************** Section 1 Introduction ***********************-->
+## Introduction
+
+Tomviz visualizes the resulting data set at the end of each processing pipeline, where normally several operations have been applied. However, if no operators have been applied, the original data will be visualized.
+
+Users can also display the original data directly from the beginning of a pipeline, by simply selecting the data set, and adding modules to it. Data sets can be clones, which means creating a new root.
+
+During the process, Tomviz saves the pipeline every five minutes. This can be recovered upon next restart of Tomviz. Users can also save the application state by taking a snapshot of the pipeline at a given moment, which can be restored, edited, saved again later. Note that saving state does not save data.
+
+![tomviz_start_end_pipeline](img/tomviz_start_end_pipeline.png)
+
+Typical datasets may have following relationships between attributes and sizes.
+
+| Volume          | Voxels        | Size (char) | Size (int) |
+|  :---           |  :---         |    :---     | :---       |
+| 64<sup>3</sup>  | 262,144       | 0.25 MB     | 1 MB       |
+| 128<sup>3</sup> | 2,097,152     | 2.00 MB     | 8 MB       |
+| 256<sup>3</sup> | 16,777,216    | 16.00 MB    | 64 MB      |
+| 512<sup>3</sup> | 134,217,728   | 128.00 MB   | 512 MB     |
+| 103<sup>3</sup> | 1,073,741,824 | 1,024.00 MB | 4096 MB    |
+
+### Load data
+
+In this subsection, loading methods for three data categories, single data file, stack of images and raw data set, are being introduced.
+
+#### Single data file
+
+Loading single data set is straight forward, simply select ```Open Data``` from the ```File``` menu, as indicated in screenshot below.
+
+![tomviz_open_data](img/tomviz_open_data.png)
+
+#### Image stacks
+
+Loading image stacks takes a little bit more efforts than loading single data file. After selecting ```Open Stack``` from the ```File``` menu, check all the images you would like to include in the pop-up window, as indicated in screenshots below.
+
+![tomviz_open_stack](img/tomviz_open_stack.png)
+
+![tomviz_stack_dialog](img/tomviz_stack_dialog.png)
+
+#### Reading a raw file
+
+Users can also choose to read raw files by defining data dimensions, type, endianness, and etc, as indicated below.
+
+![raw_reader](img/raw_reader.png)
+
+### Save results
+
+#### Save data
+
+Users can save the data by either clicking the ```Save Data``` button from ```File``` (as shown below), or simply using the keyboard short-cut ```Ctrl+S```.
+
+![tomviz_save_data](img/tomviz_save_data.png)
```

Comment:
In general it is a shame you didn't carry over the alt text from the slides, these all appear to be the file name with out the suffix. We can add these back in and maybe improve them in a future iteration.

Reply:
Oh I did not know those names within the brackets were there for a reason, I will copy them over then. Thanks.

Reply:
Just to make sure, the `alt text` you mean the text inside the brackets right?

Reply:
Yes - `![Save data](./images/tomviz_save_data.png)` where `[Save data]` is the image alt text, and `(/images/tomviz_save_data.png)` is the path to the image.